site/layouts/partials/docs-navbar.html

```diff
@@ -26,8 +26,8 @@
     </ul>
   </div>
 
-  <ul class="navbar-nav flex-row ml-md-auto d-none d-md-flex">
-    <li class="nav-item dropdown">
+  <ul class="navbar-nav ml-md-auto mt-3">
```

Comment:
These changes could be separated, so I think I'm going to make a PR against master later.